src/layouts/SettingsMenu.tsx

```diff
@@ -69,7 +69,7 @@ export function SettingsMenu() {
         </MenuItem>
 
         <DarkModeSwitcher component={MenuItem} />
-        <TestNetModeSwitcher component={MenuItem} />
+        {process.env.NEXT_PUBLIC_ENV === 'app' && <TestNetModeSwitcher />}
```

Comment:
hmm should we name this "app" or something like prod or production

Reply:
Yeah that makes more sense, I'll update it

Reply:
Changing it to prod